### ISSUE TITLE
Steam - Cloud Sync

### ISSUE THREAD

**Issue by Async0x42** [aseprite/aseprite#953]
For the Steam version, cloud sync would be great so that our settings would transfer across multiple devices (rather than using the user home directory)

**Comment by dacap** [comment]
I though about it. I'm not sure if all settings should be sync or if there should be a list of groups to be synced (e.g. maybe we don't want to sync the "zoom with mouse wheel" because we use the trackpad on OS X, and a mouse on a PC desktop)

**Comment by vncastanheira** [comment]
Do you use the same setting to do different things across OS?

**Comment by dacap** [comment]
No, it's the same setting. Maybe I should see if I can differentiate a mouse wheel event (apple magic touch event) from the trackpad touch.

**Comment by Nsheppard01** [comment]
What about having steam cloud sync saved images?

**Comment by Amoo0001** [comment]
Syncing extensions and user themes, etc would also be a real help.  I lost a custom theme I worked quite hard on when my old laptop died because I forgot to back it up.
